app/jobs/preserve_resource_job.rb

```diff
@@ -6,7 +6,8 @@ class PreserveResourceJob < ApplicationJob
 
   # @param id [String] resource id
   # @param lock_tokens [Array<String>] serialized lock tokens
-  def perform(id:, lock_tokens: nil)
+  # @param force_preservation [Boolean] force binary node preservation
+  def perform(id:, lock_tokens: nil, force_preservation: nil)
```

Comment:
Suggestion: Default to false instead of nil.

Reply:
Good idea. 👍